src/screens/Login/LoginForm.tsx

```diff
@@ -215,6 +227,36 @@ export const LoginForm = ({
           </TextField.Root>
         </View>
       </View>
+      {isAuthFactorTokenNeeded && (
+        <View>
+          <TextField.LabelText>
+            <Trans>2FA Confirmation</Trans>
+          </TextField.LabelText>
+          <TextField.Root>
+            <TextField.Icon icon={Ticket} />
+            <TextField.Input
```

Comment:
Pressing Enter while in this field doesn't submit the form which feels annoying

Reply:
Fixed